pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ata-db-models"
-version = "0.0.16"
+version = "0.0.17"
```

Comment:
🔧 version bump not needed because this is an internal change; only need a version bump when we make changes that would be used by any dependencies (so if we changed the models, for example)

Reply:
Ah cool! Reverting now